.palantir/revapi.yml

```diff
@@ -1100,6 +1100,12 @@ acceptedBreaks:
     - code: "java.class.removed"
       old: "enum org.apache.iceberg.BaseMetastoreTableOperations.CommitStatus"
       justification: "Removing deprecated code"
+    - code: "java.method.parameterTypeChanged"
```

Comment:
unfortunately we can't just break APIs. You would have you create an additional method and deprecate the old one

Reply:
I've updated the deprecation message but the new version looks much better, thanks